model/components/thanos.yml

```diff
@@ -0,0 +1,33 @@
+---
+components:
+
+  # thanos
+  - id: thanos
+    name: Thanos
+    description: Thanos is an CNCF project, set of components that can be composed into a highly available metric system with unlimited storage capacity, which can be added seamlessly on top of existing Prometheus deployments.
+    link: https://thanos.io/
+    github:
+      - thanos-io/thanos
+    twitter: ThanosMetrics
+    logo: thanos
+    license:
+      - Apache License 2.0
+    categories:
+      - visualization
+      - storage
+    connections:
+      dataFrom:
+        - prometheus-server
+        - open-tsdb
+      dataTo:
```

Comment:
I assume some of the 'dataTo' components are listed here as kind of "self-monitoring" of Thanos (e.g. all the tracing related tools), right?

In the Landscape we try to focus only on monitoring data flows with respect to the target systems. So we explicitly exclude self-monitoring aspects (monitoring the monitoring system), as otherwise the model would get to complicated and confusing. Could you fix this please, then we can merge!

Reply:
Ah, I see! Reduced to data flow then (: